boardswarm/src/main.rs

```diff
@@ -239,19 +239,31 @@ trait DeviceConfigItem {
 }
 
 impl DeviceConfigItem for config::Console {
+    #[instrument(fields(name = self.name), skip(self, properties))]
```

Comment:
```suggestion
    #[instrument(fields(name = self.name), skip_all, level="error")]
```
set the span to the highest verbosity so it's always included on output (by default it's info level). Ofcourse alternatively given this is such a small function may make sense to just put the name in the message directly.

I do wonder if this wouldn't be better in `Device::from_config` as a check on device creation (so it dumps all the warnings just once) as opposed to checks in `matches` which would warns every time a match is checked.

Reply:
Well you'll be seeing the behaviour of it potentially spamming the log :)  An empty match is really surprising though such i don't really mind either way